.lintrunner.toml

```diff
@@ -1544,7 +1544,6 @@ exclude_patterns = [
     'test/test_namedtuple_return_api.py',
     'test/test_native_functions.py',
     'test/test_native_mha.py',
-    'test/test_nestedtensor.py',
```

Comment:
Given that other test files are skipped, why auto-format this one?

Reply:
For this PR, I just want to auto format the whole file before I make other logic changes so they won't be mixed together. 
Should all tests ignored by UFMT linter? If that's the case, I can add it back to the exclude list.